README.md

```diff
@@ -1,4 +1,6 @@
-# Code Climate PHP_CodeSniffer Engine
+# Code Climate PHP_CodeSniffer Engine<br>
+
+[![Code Climate](https://codeclimate.com/repos/558b317f69568073bf0021e5/badges/7c2a59b5bee6d2425207/gpa.svg)](https://codeclimate.com/repos/558b317f69568073bf0021e5/feed)
```

Comment:
This repo should be added as an open source repo and then we should use that badge (so that it's accessible to everyone)